### DISCUSSION TITLE
Maintainers TODO list

### DISCUSSION THREAD

**Discussion by EnricoMi** [PyGithub/PyGithub#2453] [General]
I understand that [MAINTAINERS](https://github.com/PyGithub/PyGithub/blob/master/MAINTAINERS) of this project have little time to support this project. In order to dedicate the time they have into the activities with most outcome, I'd like to steer their attention to the most valuable next steps.

When maintainers find time, please go through these TODOs from top to bottom.

Release 1.58.0 is broken for many users, please approve these fixes:
(I cannot approve them myself because I have created those fixes):
- #2446
- #2443
- #2439
- #2460

Once those are approved, I'd like to release 1.58.2.

Small fixes to annoying bugs:
- #2159

Pull requests with impact:
- #2145
- #2387

Most wanted pull requests as voted by the community:
Open: https://github.com/PyGithub/PyGithub/pulls?q=is%3Apr+is%3Aopen+sort%3Areactions-%2B1-desc
Stale: https://github.com/PyGithub/PyGithub/pulls?q=is%3Apr+sort%3Areactions-%2B1-desc+is%3Aclosed+label%3Astale

**Comment by EnricoMi** [comment]
This is a locked discussion as this is to communicate with maintainers to keep noise at a minimum so they can focus on the tasks.

If someone likes to get maintainers attention, please react on the **_description_** of an issue or pull request with a :+1:.

That's the best you can do. Details in https://github.com/PyGithub/PyGithub/issues/2178#issuecomment-1263184406.

**Reply by s-t-e-v-e-n-k** [reply]
Thanks for doing this! I had a small amount of time so I've moved this along, but please prod me here when my comment has been addressed and I'll try and find some time to cut a new release.

**Reply by EnricoMi** [reply]
addressed...

**Reply by EnricoMi** [reply]
@s-t-e-v-e-n-k there is another fix that should go into 1.58.1 (see above).

Is there a chance to make release a one-click thing on Gitthub? How do you feel about handing the release process on?

The [v1.58.1](https://github.com/PyGithub/PyGithub/releases/tag/v1.58.1) hotfix tag has been sitting there for a month, waiting to be released. I can see you have prepared the release draft, and it looks like I have permissions to submit it. What else is needed for a release?

**Reply by EnricoMi** [reply]
It looks like that v1.58.1 version has been released to PyPi. I did not recognize this has happened.

What do we do with https://github.com/PyGithub/PyGithub/pull/2460 then, releasing as v1.58.2?

**Reply by s-t-e-v-e-n-k** [reply]
The release process involves pushing to the master branch, which I can not do, since it requires an approved PR. I was trying to work out what to do here when I ran out of time, and I've not had any free time to circle back to it.
